src/edit/DistortableImage.Edit.js

```diff
@@ -69,7 +77,8 @@ L.DistortableImage.Edit = L.Handler.extend({
 	},
 
 	/* Run on image deseletion. */
-	removeHooks: function() {
+	removeHooks: function ()
+	{
```

Comment:
This is looking really nice! Actually we kind of prefer in-line syntax for these -- `function() {` -- thanks!

Reply:
I can fix that - I'm still in that habit from what I do with C# - I think VS might have done it for me.